### PR TITLE
unused local variable is removed

### PR DESCRIPTION
**Pull request by gozeloglu**
Signed-off-by: gozeloglu <gozeloglu@gmail.com>
Change-Id: Ia44a715af089cc6c09f51e8aba3be8be0b3ff31a

* Resolves: # 
* Target version: master 

### Summary
I removed the unused local variable with respect to https://lgtm.com/projects/g/CollaboraOnline/online/rev/4c5bf4f116bdfb7cb77df18c8d55391fb19ce7cf . 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: loleaflet/src/control/Control.JSDialogBuilder.js [hunk 1]
```diff
@@ -2482,7 +2482,6 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			if (twoPanelsAsChildren) {
 				handler = this._controlHandlers['paneltabs'];
-				processChildren = handler(childObject, childData.children, this);
```
Comment (Ashod) [inline loleaflet/src/control/Control.JSDialogBuilder.js:2485]: Doesn't handler() have side-effects? I expect it does.

Notice that by removing the call the variable 'handler' is not user after the assignment.

**Review by Ashod — COMMENTED** [review]
Thanks @gozeloglu for the patch!

I see you removed the call to handler() as well as the local variable. Is the call unnecessary or did you intend to remove the local variable processChildren only?

Thanks again for your contibution.

File: loleaflet/src/control/Control.JSDialogBuilder.js (hunk 1 above)
Comment (gozeloglu) [reply loleaflet/src/control/Control.JSDialogBuilder.js:2485]: I just want to remove variable. Actually, I did not look at side-effects.

Comment (Ashod) [reply loleaflet/src/control/Control.JSDialogBuilder.js:2485]: Perhaps best to restore the handler call?

**Comment by mrkara** [comment]
@gozeloglu A polite ping. Do you need help to amend this patch? :)

**Comment by gozeloglu** [comment]
> @gozeloglu A polite ping. Do you need help to amend this patch? :)

I did not have enough time to look at the issue after review. I think my changes were not so suitable for the project. If it is, you can close the PR.

**Comment by mrkara** [comment]
@gozeloglu Thank you a lot for the effort! :)

Please feel free to grab an [easy hack](https://github.com/CollaboraOnline/online/issues?q=is%3Aissue+is%3Aopen+label%3A%22Easy+Hack%22) or [an open issue](https://github.com/CollaboraOnline/online/issues) to continue. Looking forward to your next PR! ;)

**Pull request closed without merging**